src/SignalingClient.spec.ts

```diff
@@ -572,4 +572,30 @@ describe('SignalingClient', () => {
             });
         });
     });
+
+    describe('outsideBrowser', () => {
+        it('parseJSONObjectFromBase64String', done => {
+            global.atob = undefined;
+            const client = new SignalingClient(config as SignalingClientConfig);
+            client.once('sdpAnswer', (sdpAnswer, _) => {
```

Comment:
Can we get rid of _ to fix these warnings?

Reply:
Done